pkg/mutate/server.go

```diff
@@ -62,15 +62,16 @@ func NewMutationServer(ctx context.Context, c *MutationConfig) (*MutationServer,
 func (s *MutationServer) handleMutate(w http.ResponseWriter, r *http.Request) {
 	// read the body / request
 	body, err := ioutil.ReadAll(r.Body)
-
 	if err != nil {
+		err = fmt.Errorf("unable to correctly read the body of the request")
```

Comment:
Would log the actual error, for the sake of troubleshooting. Same below

Reply:
Sure, thank you